integration/src/test/resources/ammonite/integration/basic/Spark2.json

```diff
@@ -1,6 +1,4 @@
-[
-  {"hello": 1},
-  {"hello": 2},
+{"hello": 1},
+{"hello": 2},
 
-  {"hello": 3}
-]
+{"hello": 3}
```

Comment:
That's how Spark JSON reader works by default. With the current Spark2.json the result of `df` would be:
```
+---------------+-----+
|_corrupt_record|hello|
+---------------+-----+
|              [| null|
|           null|    1|
|           null|    2|
|           null|    3|
|              ]| null|
+---------------+-----+
```

Now it produces the following:
```
+-----+
|hello|
+-----+
|    1|
|    2|
|    3|
+-----+
```
The official Spark documentation uses the same format for their examples.